statistics/handle/handle.go

```diff
@@ -221,6 +221,155 @@ func (h *Handle) Update(is infoschema.InfoSchema) error {
 	return nil
 }
 
+// GlobalStats is used to store the statistics contained in the global-level stats
+// which is generated by the merge of partition-level stats.
+// It will both store the column stats and index stats.
+// In the column statistics, the variable `num` is equal to the number of columns in the partition table.
+// In the index statistics, the variable `num` is always equal to one.
+type GlobalStats struct {
+	num   int
+	count int64
+	hg    []*statistics.Histogram
+	cms   []*statistics.CMSketch
+	topN  []*statistics.TopN
+}
```

Comment:
How about exposing this struct's fields and removing its methods, which may make it clearer?